<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListViewMainWidget</class>
 <widget class="QWidget" name="ListViewMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#headerWidget{
 background-color:#061822;
}

QWidget#leftFilterWidget{
 background-color:#061D2C;
}

QLabel#title{
color:#D2EFFC;
}

QWidget#rightListWidget
{
  background-color:#000000;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#searchBtn{
   background-color:#043B4E;
   color:#ffffff;
   min-height:30px;
   max-height:30px;
   min-width:90px;
   max-width:90px;
   border-radius:0px;
}

QLineEdit#lineEidt{
  min-height:30px;
  max-height:30px;
  min-width:100px;
  max-width:300px;
  background-color: #172125;
  color: #5E5F5F;
}

QLabel#lableIcon{
      min-width:25px;
      max-width:25px;
      min-height:30px;
      max-height:30px;
      background-color: #172125;
}

QWidget#iconWidget{
      min-width:25px;
      max-width:25px;
      min-height:30px;
      max-height:30px;
      background-color: #172125;
      border-right:0px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CTA冠脉斑块标注平台</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="iconWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelIcon">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/source/search .svg</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CustomLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #172125;
color: #5E5F5F;</string>
        </property>
        <property name="text">
         <string>输入序列号搜索相关文件</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchBtn">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="leftFilterWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#uploadBtn{
	font: 11pt &quot;黑体&quot;;
  min-width:120px;
  max-width:120px;
  min-height:40px;
  max-height:40px;
  margin-left:40px;
  border:0px solid ;
  border-radius:5px;
  background-color:#448E97;
  color:#ffffff;
}

QPushButton#importFinishedBtn{
  font: 10pt &quot;黑体&quot;;
  min-width:120px;
  max-width:120px;
  min-height:40px;
  max-height:40px;
  margin-left:40px;
  border:0px solid ;
  border-radius:5px;
  background-color:#043B4E;
  color:#ffffff;
}

QWidget#filterBtnWidget
{
  background-color:#000000;
}




</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,4,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>73</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="uploadBtn">
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>73</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QWidget" name="filterBtnWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>160</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:hover{
  background-color:#043B4E;
  font: 11pt &quot;黑体&quot;;
  color:#ffffff;
  padding:0px;;
    border:none;
}

QPushButton{
  border:none;
  min-height:40px;
  max-height:40px;
  font: 10pt &quot;黑体&quot;;
  background-color:#061D2C;
  color:#666666; 
}

/*QPushButton:press{
  padding-left:0px;  
  padding-top:0px; 
    border:none; 
}*/</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="selectAllBtn">
               <property name="text">
                <string>全部文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="finishedBtn">
               <property name="text">
                <string>已完成标注文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="annoingBtn">
               <property name="text">
                <string>标注中文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unAnnoBtn">
               <property name="text">
                <string>未标注文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="importFinishedBtn">
            <property name="text">
             <string>导出完成序列</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>73</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="rightListWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="layoutListWid">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
